vars/npmInstall.groovy

```diff
@@ -9,6 +9,13 @@ def call(body) {
     sh '''
         npm install --production
         npm ls
+      '''
+
+    // Check the licenses before the dev dependencies are installed. We don't ship them in
+    // the artifacts so we don't have to include their licenses
+    checkLicenses
```

Comment:
I think it it'd be more efficient to just inline the 2 shell commands here in the one `sh` block. We've seen cases recently where having lots and lots of `sh` blocks can sometimes cause things to take a lot of time. Not really sure why yet. Could you inline it?

Reply:
Yes actually there's no need for an extra file. Will update.